index.html

```diff
@@ -1,37 +1,192 @@
+                <p class="text2">Back End Dev</p>
+                <img class="dot" src="Counter.png" alt="counter">
+                <p class="text2">2015</p>
+            </section>
+            <p class="cardText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <section class="cards">
+                <p class="firstcard">HTML</p>
+                <p class="firstcard">CSS</p>
+                <p class="firstcard">javascript</p>
+            </section>
+            <section>
+                <p class="buttonProject">See Project</p>
+            </section>
+        </section>
+        <section class="content3">
+            <img class="snapshot" src="SnapshootPortfolio(4).png" alt="picture">
+            <h1 class="text">Tonic</h1>
+            <section class="block">
+                <p class="text1">CANOPY</p>
+                <img class="dot" src="Counter.png" alt="counter">
+                <p class="text2">Back End Dev</p>
+                <img class="dot" src="Counter.png" alt="counter">
+                <p class="text2">2015</p>
+            </section>
+            <p class="cardText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <section class="cards">
+                <p class="firstcard">HTML</p>
+                <p class="firstcard">CSS</p>
+                <p class="firstcard">javascript</p>
+            </section>
+            <section>
+                <p class="buttonProject">See Project</p>
+            </section>
+        </section>
+        <section class="content4">
+            <img class="snapshot" src="SnapshootPortfolio(5).png" alt="picture">
+            <h1 class="text">Multi-Post <br> Stories</h1>
+            <section class="block">
+                <p class="text1">CANOPY</p>
+                <img class="dot" src="Counter.png" alt="counter">
+                <p class="text2">Back End Dev</p>
+                <img class="dot" src="Counter.png" alt="counter">
+                <p class="text2">2015</p>
+            </section>
+            <p class="cardText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <section class="cards">
+                <p class="firstcard">HTML</p>
+                <p class="firstcard">CSS</p>
+                <p class="firstcard">javascript</p>
+            </section>
+            <section>
+                <p class="buttonProject">See Project</p>
+            </section>
+        </section>
+    </section>
+
+
+
+    <section class="about">
+        <h1 class="nameAbout">About<br>Myself</h1>
+        <p class="introAbout">Hello I'm a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded, don't
+            hestiate to contact me.</p>
+        <p class="touch">LET'S CONNECT</p>
```

Comment:
- [ ] We can switch this to using one of the heading tags. a `hx` tag instead of a `p` tag will signify this piece of text as more important than more generic text in a `p` tag.
```suggestion
        <h3 class="touch">LET'S CONNECT</h3>
```